### ISSUE TITLE
Reminders

### ISSUE THREAD

**Issue by nyarly** [nyarly/simplekey#8]
Keep track of the last time that the user performed certain actions, issue a reminder to do them.

Things like refreshing keys, running "advise"

Maybe key refresh should be automatic?

**Comment by strugee** [comment]
Of course, that implies the question: what is the "right" way to refresh keys? Some (including myself) would say Parcimonie.

**Comment by strugee** [comment]
FWIW, I believe there was a bash port of Parcimonie written at some point.

**Comment by dashohoxha** [comment]
Parcimonie can be installed with `apt-get install parcimonie` (on Debian based systems). I think that it should be part of setting up (or initializing) the system, and that's all.

By the way, I have started a fork of the project here: https://github.com/dashohoxha/egpg
I don't know all the details about GnuPG (it is huge), so any ideas or suggestions about improving it would be wellcome.
